README.md

```diff
@@ -57,3 +57,9 @@ Then run tests with nose:
 ```commandline
     nosetests
 ```
+
+You can run nosetests with python3 using:
+
+```commandline
+    python3 -m "nose" [test file]
```

Comment:
could you mention that you need to add `test/...` before the test file name

Reply:
added clarification